src/Crowdfunding/N3O.Umbraco.Crowdfunding/Controllers/CrowdfundingController.Content.cs

```diff
@@ -10,16 +10,8 @@ namespace N3O.Umbraco.Crowdfunding.Controllers;
 public partial class CrowdfundingController {
     [HttpGet("content/{contentId:guid}/properties/{propertyAlias}")]
     [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<ActionResult<ContentPropertyValueRes>> GetContentPropertyValue() {
-        var res = await _mediator.Value.SendAsync<GetContentPropertyValueQuery, None, ContentPropertyValueRes>(None.Empty);
-
-        return Ok(res);
-    }
-    
-    [HttpGet("content/{contentId:guid}/nested/{propertyAlias}/schema")]
-    [ProducesResponseType(StatusCodes.Status404NotFound)]
-    public async Task<ActionResult<NestedPropertySchemaRes>> GetNestedPropertySchema() {
-        var res = await _mediator.Value.SendAsync<GetNestedPropertySchemaQuery, None, NestedPropertySchemaRes>(None.Empty);
+    public async Task<ActionResult<ContentPropertyRes>> GetContentPropertyValue() {
+        var res = await _mediator.Value.SendAsync<GetContentPropertyValueQuery, None, ContentPropertyRes>(None.Empty);
```

Comment:
I would leave the GetNestedPropertySchema API, it will be useful in future if not now